Gruntfile.js

```diff
@@ -35,7 +35,7 @@ module.exports = function (grunt) {
 
 		mochaIstanbul : {
 			coverage : {
```

Comment:
I noticed you created a `test/spec` directory. Maybe your intent is to have `test/helpers` directory for helpers. Either way, using the mask wouldn't hurt.